graphics/include/ignition/common/SubMesh.hh

```diff
@@ -124,6 +124,20 @@ namespace ignition
       /// \param[in] _uv The texture coordinate
       public: void AddTexCoord(const ignition::math::Vector2d &_uv);
```

Comment:
How about documenting what set the set-less APIs add to? I noticed it defaults to zero, but may be different if another set has started.

Reply:
added more documentation in 228eb7c